deploy/core-agents-deployment.yaml

```diff
@@ -23,8 +23,7 @@ spec:
         - command:
           - sh
           - -c
-          - trap "exit 1" TERM; until nc -vz mayastor-etcd 2379; do echo "Waiting for etcd...";
-            sleep 1; done;
+          - trap "exit 1" TERM; until nc -vz mayastor-etcd 2379; do echo "Waiting for etcd..."; sleep 1; done;
```

Comment:
I'm not sure why these lines changed - I didn't touch the helper that generates this?